packages/dappmanager/src/modules/nsupdate/utils.ts

```diff
@@ -1,6 +1,6 @@
 import { isEmpty } from "lodash";
 import { PackageContainer } from "../../types";
-import { getContainerDomain } from "../../params";
+import params, { getContainerDomain } from "../../params";
```

Comment:
these are functional changes, not typos must be split to multiple PRs and have an associated Issue